client/my-sites/stats/activity-log/index.jsx

```diff
@@ -192,49 +193,58 @@ class ActivityLog extends Component {
 		</div>;
 	}
 
-	// FIXME: This is for internal testing
 	renderErrorMessage() {
 		const {
 			isPressable,
 			rewindStatusError,
 			translate,
 		} = this.props;
 
-		// FIXME: Do something nicer with the error
+		// Do not match null
+		// FIXME: This is for internal testing
+		if ( false === isPressable ) {
+			return (
+				<ActivityLogBanner status="info" icon={ null } >
+					{ translate( 'Rewind is currently only available for Pressable sites' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Currently only available for Pressable sites' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).